server/game/cards/07.1-DDeeds/PersonalOrnithopter.js

```diff
@@ -0,0 +1,37 @@
+const GameActions = require('../../GameActions/index.js');
+const GoodsCard = require('../../goodscard.js');
+
+class PersonalOrnithopter extends GoodsCard {
+    setupCardAbilities(ability) {
+        this.action({
+            title: 'Repeat Shootout: Join Posse',
+            playType: 'shootout:join',
+            cost: ability.costs.payGhostRock(1),
+            condition: () => this.game.shootout,
```

Comment:
```suggestion
            condition: () => this.game.shootout && !this.parent.isParticipating(),
```
Make sure the flying dude is not participating already.